### PR TITLE
CSHARP-2656: Database-level aggregation should be tested on server versions 3.6+

### PR DESCRIPTION
**Pull request by vincentkam**
https://jira.mongodb.org/browse/CSHARP-2656
Spec synced from: https://github.com/mongodb/specifications/commit/916f85424c784728827cba2f56e90cd62a6b0d57
https://evergreen.mongodb.com/version/5d2382262fbabe24adec32ec

### CODE REVIEW

File: tests/MongoDB.Driver.Tests/Specifications/crud/tests/read/db-aggregate.json [hunk 1]
```diff
@@ -1,148 +1,73 @@
 {
   "database_name": "admin",
-  "minServerVersion": "4.0",
+  "minServerVersion": "3.6",
```
Comment (vincentkam) [inline tests/MongoDB.Driver.Tests/Specifications/crud/tests/read/db-aggregate.json:3]: Synced spec from https://github.com/mongodb/specifications/commit/916f85424c784728827cba2f56e90cd62a6b0d57

**Comment by vincentkam** [comment]
Ready for review

**Review by DmitryLukyanov — APPROVED** [review]
LGTM

**Review by ksadoff — APPROVED** [review]
LGTM

**Review by rstam — APPROVED** [review]

**Comment by jyemin** [comment]
@vincentkam please close this.

**Comment by vincentkam** [comment]
Whoops, sorry. Closing as this was merged to master already

**Pull request closed without merging**